File: Assignment-33/Program4.java
import java.util.*;

class Program4
{
      public static void main(String arg[])
      {
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter The Number:");
            int iNo = sobj.nextInt();

            Digit obj = new Digit();

            int iRet = obj.Multiply(iNo);
            System.out.println(iRet);
      }
}
class Digit
{
      public int Multiply(int iNo)
      {
            int iMult = 1;
            int iDigit = 0;

            if(iNo < 0)
            {
                  iNo = -iNo;
            }
            while(iNo !=0)
            {
                  iDigit = iNo % 10;
                
                  iMult = iMult * iDigit;

                  iNo = iNo / 10;
            }
            return iMult;

      }
}

File: Assignment-13/program4.c
#include<stdio.h>

int MultDigit(int iNo)
{
      int iDigit = 0;
      
      int iMult = 1;

      if(iNo < 0)
      {
            iNo =-iNo;
      }
      while(iNo > 0)
      {
            iDigit = iNo % 10;

            iMult = iMult * iDigit;

            iNo = iNo / 10;
      }
      return iMult;
      
}
int main()
{
      int ivalue = 0;
      int iRet = 0;

      printf("Enter The Number :");
      scanf("%d",&ivalue);

      iRet = MultDigit(ivalue);

      printf("%d",iRet);

      return 0;
}